benchmarks/ampi/alltoall/Makefile

```diff
@@ -1,9 +1,10 @@
+-include ../../common.mk
 OPTS = -O3
 CHARMC = ../../../bin/ampicc
-CHARMF = ../../../bin/ampif90
 MPICC = mpicc
 
-all: mpibench mpibench.mpi
+all: mpibench allgather alltoall alltoall_VPtest
+all-mpi: mpibench.mpi allgather.mpi alltoall.mpi
```

Comment:
Do `allgather.mpi alltoall.mpi` exist?

Reply:
Didn't entirely get you. 

They are valid targets in the Makefile. If you're asking if they run successfully, they do. 
```

nbhat4@courage:/scratch/nitin/charm/mpi-linux-x86_64-debug/benchmarks/ampi/alltoall$ make test-mpi
mpirun -n 4 ./mpibench.mpi
#Alltoall: P=4, NUMTIMES=1, REPS=2000
4 2.7415 2.84 3.0295 2.805 3.761 15.8545 36.5685 69.284 880.196
mpirun -n 4 ./allgather.mpi 1024
Starting benchmark on 4 processors with 100 iterations
  100    1024	    0.004 msec,	 11377.396 Mbits/sec
mpirun -n 4 ./alltoall.mpi 1024
Starting benchmark on 4 processors with 1000 iterations
 1000    1024	    0.004 msec,	 4625.627 Mbits/sec
```

Reply:
Ah, I missed the definition of their build targets. Should they be added to the `clean` target too?